source/includes/fundamentals/code-snippets/MongoDbAwsAuth.java

```diff
@@ -73,22 +92,35 @@ private static void mongoCredentialSessionTokenCredential() {
         // end mongoCredentialSessionTokenCredential
     }
 
+    private static void mongoCredentialECS() {
+        // start mongoCredentialECS
+        MongoCredential credential = MongoCredential.createAwsCredential(null, null).withMechanismProperty("AWS_CONTAINER_CREDENTIALS_RELATIVE_URI", "<your ECS endpoint>");
```

Comment:
`AWS_CONTAINER_CREDENTIALS_RELATIVE_URI ` is an environment variable, not a mechanism property

Reply:
Sorry for the confusion. For some reason, I thought this was how you specified the relative URI when using MongoCredentials. From a later comment, does this mean that you cannot use container credentials if you specify credentials using MongoCredential.createAwsCredential()?

Reply:
No, it doesn't mean that.  As long as you pass `null, null` as the parameters, it will use either env vars or container credentials